zstacklib/zstacklib/scripts/rbd_rw.py

```
@@ -0,0 +1,237 @@
+        time.sleep(0.05)
+    if not completion.is_complete():
+        raise Exception("write image timeout after 5 seconds")
+    completion.wait_for_complete_and_cb()
+    if completion.get_return_value() != len(content):
+        raise Exception("write image not completed, content length: {} return code: {}".format(
+            len(content), completion.get_return_value()))
+
+
+def listen_pipe():
+    while True:
+        operation = raw_input()
```

Ruff (0.8.2)

Undefined name `raw_input`

(F821)